### ISSUE TITLE
ValueError: Could not find Jackhmmer database /mnt/mgnify_database_path/mgy_clusters_2018_08.fa

### ISSUE THREAD

**Issue by hradhakrishnan** [deepmind/alphafold#54]
The download script for mgnify downloaded and created file, mgy_clusters.fa , however couldn't locate mgy_clusters_2018_08.fa which seems to be referenced by /app/alphafold/alphafold/data/tools/jackhmmer.py

```
packages/absl/flags/_validators.py:203: UserWarning: Flag --preset has a non-None default value; therefore, mark_flag_as_required will pass even if flag is not specified in the command line!
I0724 21:57:01.485210 139819066255168 run_docker.py:193] warnings.warn(
I0724 21:57:02.218386 139819066255168 run_docker.py:193] I0724 21:57:02.217257 140387317352256 templates.py:880] Using precomputed obsolete pdbs /mnt/obsolete_pdbs_path/obsolete.dat.
I0724 21:57:02.224482 139819066255168 run_docker.py:193] E0724 21:57:02.223666 140387317352256 jackhmmer.py:65] Could not find Jackhmmer database /mnt/mgnify_database_path/mgy_clusters_2018_08.fa
I0724 21:57:02.225311 139819066255168 run_docker.py:193] Traceback (most recent call last):
I0724 21:57:02.225568 139819066255168 run_docker.py:193] File "/app/alphafold/run_alphafold.py", line 283, in <module>
I0724 21:57:02.225908 139819066255168 run_docker.py:193] app.run(main)
I0724 21:57:02.226118 139819066255168 run_docker.py:193] File "/opt/conda/lib/python3.8/site-packages/absl/app.py", line 312, in run
I0724 21:57:02.226256 139819066255168 run_docker.py:193] _run_main(main, args)
I0724 21:57:02.226412 139819066255168 run_docker.py:193] File "/opt/conda/lib/python3.8/site-packages/absl/app.py", line 258, in _run_main
I0724 21:57:02.226542 139819066255168 run_docker.py:193] sys.exit(main(argv))
I0724 21:57:02.226696 139819066255168 run_docker.py:193] File "/app/alphafold/run_alphafold.py", line 218, in main
I0724 21:57:02.226851 139819066255168 run_docker.py:193] data_pipeline = pipeline.DataPipeline(
I0724 21:57:02.226989 139819066255168 run_docker.py:193] File "/app/alphafold/alphafold/data/pipeline.py", line 104, in __init__
I0724 21:57:02.227127 139819066255168 run_docker.py:193] self.jackhmmer_mgnify_runner = jackhmmer.Jackhmmer(
I0724 21:57:02.227246 139819066255168 run_docker.py:193] File "/app/alphafold/alphafold/data/tools/jackhmmer.py", line 66, in __init__
I0724 21:57:02.227406 139819066255168 run_docker.py:193] raise ValueError(f'Could not find Jackhmmer database {database_path}')
I0724 21:57:02.227567 139819066255168 run_docker.py:193] ValueError: Could not find Jackhmmer database /mnt/mgnify_database_path/mgy_clusters_2018_08.fa
```

**Comment by xiaoyongzhu** [comment]
You might want to copy and rename `mgy_clusters.fa` to `mgy_clusters_2018_08.fa`. This seems to be a small bug since Deepmind have updated some of the names of the files

**Comment by miesav** [comment]
Ran into this as well. This worked so far:

ln -s mgy_clusters_2018_12.fa mgy_clusters_2018_08.fa

**Comment by joemin** [comment]
it looks like the provided download script downloads mgy_clusters_2018_12.fa instead of mgy_clusters_2018_08.fa. DeepMind should probably either avoid public-facing versioned naming or update their scripts.

a fix in addition to the one that @miesav provided is to change line 60 in docker/run_docker.py from
`    DOWNLOAD_DIR, 'mgnify', 'mgy_clusters_2018_08.fa')`
to
`    DOWNLOAD_DIR, 'mgnify', 'mgy_clusters_2018_12.fa')`
which worked for me!

**Comment by joemin** [comment]
PR is open for this now:
#68

**Comment by tfgg** [comment]
Fixed by 20a705778fc0315ffce93878582b2cfc3836ae81 - apologies for the typo.
